searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/UnzipFile.java

```diff
@@ -15,6 +15,13 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipInputStream;
 import java.nio.file.Path;
+
+/**
+ * It is responsible for analyze an image and detect if there it's a face or not.
```

Comment:
This class has this function?, if it's not the case, please change the information

Reply:
already change